src/coverlet.core/Instrumentation/Instrumenter.cs

```diff
@@ -76,6 +77,7 @@ private void InstrumentModule()
 
                 using (var module = ModuleDefinition.ReadModule(stream, parameters))
                 {
+                    var containsAppContext = module.GetType(nameof(System), nameof(AppContext)) != null;
```

Comment:
📝 This could be modified to also check the name of the containing assembly, but it didn't _seem_ necessary.